### DISCUSSION TITLE
How about add one more config option: Force Keyboard, just like Alfred

### DISCUSSION THREAD

**Discussion by xxleyi** [johnlindquist/kit#737] [Ideas]
![image](https://user-images.githubusercontent.com/12626454/155055455-926ad561-f7d0-433f-8a1b-2965cf0cc637.png)

Since I am in China, so I always use two input methods at my daily work. However, when I lanch Kit command panel, I usually need English input method.

**Comment by johnlindquist** [comment]
This is my first time hearing about this option. I'll have to research what it would take to implement
